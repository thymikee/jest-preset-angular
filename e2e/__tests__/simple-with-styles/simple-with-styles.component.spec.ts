import { async, ComponentFixture, TestBed } from '@angular/core/testing';

import { SimpleWithStylesComponent } from './simple-with-styles.component';
import { ChildComponent } from '../medium/child.component';

describe('SimpleWithStylesComponent', () => {
  let fixture: ComponentFixture<SimpleWithStylesComponent>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      declarations: [ SimpleWithStylesComponent, ChildComponent ]
    })
    .compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(SimpleWithStylesComponent);
    fixture.detectChanges();
  });

  test('should generate snapshot without nghost/ngcontent', () => {
    expect(fixture).toMatchSnapshot();
  });
});
