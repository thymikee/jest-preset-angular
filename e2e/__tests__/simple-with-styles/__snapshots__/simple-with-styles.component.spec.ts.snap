// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SimpleWithStylesComponent should generate snapshot without nghost/ngcontent 1`] = `
<app-simple-with-styles
  __ngContext__={[Function LRootView]}
>
  <p>
     simple-with-styles works!
  
  </p><div
    class="some-class"
  >
    <app-child-component>
      <div
        class="child"
      >
          
        <div>
           rly 
          <p>
            complex
          </p>
          <div>
            component 
            <div>
              oh my god!
            </div>
          </div>
        </div>
      </div>
    </app-child-component>
  </div>
</app-simple-with-styles>
`;
